<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Client</class>
 <widget class="QWidget" name="Form_Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Клиент</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTextEdit" name="te_Log"/>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="le_Command"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pb_Send">
     <property name="text">
      <string>Отправить</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="pb_Connection">
     <property name="text">
      <string>Установить соединение</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="pb_Disconnect">
     <property name="text">
      <string>Отключить устройство</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
